grafana-plugin/src/components/CardButton/CardButton.test.tsx

```diff
@@ -1,4 +1,4 @@
-import 'jest/matchMedia.ts';
+import 'src/jest/matchMedia.ts';
```

Comment:
just curious why do we need `src` prefix here?

Reply:
At some moment it complained on that import, removed it now.